### ISSUE TITLE
cannot focus the terminal after focus out of accessible buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#175764]
https://user-images.githubusercontent.com/29464607/222216828-7d898692-64d4-4a1a-8aab-8d57a7262cb4.mov
